### PR TITLE
RELATED: SD-942 fix save button enabled

### PR DESCRIPTION
**Pull request by lanhq**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/3161
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2880

# Related Jira tasks
- SD-942: https://jira.intgdc.com/browse/SD-942

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7723/ : SUCCESS in 3m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3845/ : SUCCESS in 3s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4028/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3846/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3387/ : SUCCESS in 11m 25s

**Comment by lanhq** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1650.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3847/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3388/ : SUCCESS in 20m 05s

File: src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx [hunk 1]
```diff
@@ -59,6 +59,10 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         this.callbacks = callbacks;
         this.geoPushpinElement = element;
         this.initializeProperties(visualizationProperties);
+        // set references to null to clear references
+        this.callbacks.pushData({
+            references: this.references || null,
```
Comment (ptquang86) [inline src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx:64]: could we remove `|| null`?

Comment (lanhq) [reply src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx:64]: `null` is needed for reset the reference in extended references point. see [condition on AD](https://github.com/gooddata/gdc-analytical-designer/blob/develop/app/reducers/properties_reducer.ts#L39) for details

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4034/ : SUCCESS in 10m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/92/ : SUCCESS in 16s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7738/ : SUCCESS in 6m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1867/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2778/ : SUCCESS in 19m 16s

**Pull request merged**